README.md

```
@@ -1,41 +1,87 @@
-# WAVE API specification
+<div align="center">
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)